### PR TITLE
goodbye game

### PR DESCRIPTION
**Pull request by spookydonut**
suggestions on better places for files accepted, i wont split up files just moving them whole

https://discordapp.com/channels/498569646014857217/498576390753615882/590916594088280094

### CODE REVIEW

**Comment by optimumtact** [comment]
orange man looks on in awe as local man makes massive amounts of merge conflicts for satisfaction of own OCD

**Comment by Kmc2000** [comment]
please don't

this will destroy my tank pr even more

**Comment by spookydonut** [comment]
> 
> 
> please don't
> 
> this will destroy my tank pr even more

please learn to use rebase :)

**Comment by Rohesie** [comment]
Can you split off the changes that are not file moving for an easier review?

**Comment by spookydonut** [comment]
> 
> 
> Can you split off the changes that are not file moving for an easier review?

not really they are only because of the changed file order and make no sense otherwise.

**Comment by Rohesie** [comment]
Defines going into `__DEFINES` and changing `POD_STYLES` into a global list make sense on their own.

**Comment by spookydonut** [comment]
> 
> 
> Defines going into `__DEFINES` and changing `POD_STYLES` into a global list make sense on their own.

okay

**Comment by spookydonut** [comment]
requires #2173 first

**Comment by spookydonut** [comment]
@Rohesie pure rename pr no changes, all issues addressed, everything in a module.

File: tgmc.dme [hunk 1]
```diff
@@ -1365,6 +1272,97 @@
...
+#include "code\modules\xenomorph\mob\attack_alien.dm"
+#include "code\modules\xenomorph\mob\damage_procs.dm"
+#include "code\modules\xenomorph\mob\death.dm"
+#include "code\modules\xenomorph\mob\emote.dm"
+#include "code\modules\xenomorph\mob\evolution.dm"
+#include "code\modules\xenomorph\mob\life.dm"
+#include "code\modules\xenomorph\mob\login.dm"
+#include "code\modules\xenomorph\mob\say.dm"
+#include "code\modules\xenomorph\mob\update_icons.dm"
+#include "code\modules\xenomorph\mob\xeno_defines.dm"
```
Comment (Rohesie) [inline tgmc.dme]: Xeno mobs should be organized same as other mobs.

File: tgmc.dme [hunk 2]
```diff
@@ -1350,12 +1214,55 @@
...
+#include "code\modules\telecomms\machines\processor.dm"
+#include "code\modules\telecomms\machines\receiver.dm"
+#include "code\modules\telecomms\machines\relay.dm"
+#include "code\modules\telecomms\machines\server.dm"
+#include "code\modules\teleportation\teleportation.dm"
+#include "code\modules\teleportation\teleporter.dm"
 #include "code\modules\tgs\event_handler.dm"
 #include "code\modules\tgs\includes.dm"
 #include "code\modules\tooltip\tooltip.dm"
+#include "code\modules\toys\ashtray.dm"
```
Comment (Rohesie) [inline tgmc.dme]: Toys are just a kind of item. They should be in something like `code\modules\item\toys\` or `code\modules\obj\item\toys\`

File: tgmc.dme [hunk 3]
```diff
@@ -998,6 +780,47 @@
...
+#include "code\modules\marines\maps.dm"
+#include "code\modules\marines\memorial.dm"
+#include "code\modules\marines\mortar.dm"
+#include "code\modules\marines\overwatch.dm"
+#include "code\modules\marines\self_destruct.dm"
+#include "code\modules\marines\equipment\binoculars.dm"
+#include "code\modules\marines\equipment\marine_gear.dm"
+#include "code\modules\marines\equipment\motion_detector.dm"
+#include "code\modules\marines\equipment\portable_vendor.dm"
+#include "code\modules\marines\weapons\dropship_ammo.dm"
```
Comment (Rohesie) [inline tgmc.dme]: `marines` is a horrible module. You can put either everything or nothing it in.
Dropship stuff warrants its own module.

File: tgmc.dme (hunk 2 above)
Comment (spookydonut) [reply tgmc.dme]: https://discordapp.com/channels/498569646014857217/498576390753615882/595256059921170451

**Pull request closed without merging**